platform-tests/src/platform/availability/monitor/monitor.go

```diff
@@ -1,31 +1,33 @@
-package api_availability
+package monitor
 
 import (
 	"fmt"
+	"io"
+	"regexp"
 	"time"
 
 	"github.com/cloudfoundry-community/go-cfclient"
 )
 
-const (
-	numWorkers = 16
-)
-
 type Report struct {
-	Successes int64
-	Failures  int64
-	Errors    map[*Task]map[string]int // number of errors by error message by task
-	Elapsed   time.Duration
+	SuccessCount int64
+	FailureCount int64
+	WarningCount int64
+	Errors       map[*Task]map[string]int // number of errors by error message by task
+	Warnings     map[*Task]map[string]int // number of warnings by error message by task
+	Elapsed      time.Duration
 }
 
 func (r *Report) String() string {
+	total := r.SuccessCount + r.FailureCount + r.WarningCount
 	s := "\nReport:\n"
 	s += "==============\n"
-	s += fmt.Sprintf("Total task executions: %d\n", r.Successes+r.Failures)
-	s += fmt.Sprintf("Total Successes: %d\n", r.Successes)
-	s += fmt.Sprintf("Total failures: %d\n", r.Failures)
+	s += fmt.Sprintf("Total task executions: %d\n", total)
+	s += fmt.Sprintf("Total successes: %d\n", r.SuccessCount)
+	s += fmt.Sprintf("Total failures: %d\n", r.FailureCount)
+	s += fmt.Sprintf("Total warnings: %d\n", r.WarningCount)
 	s += fmt.Sprintf("Elaspsed time: %s\n", r.Elapsed.String())
```

Comment:
We could fix the `Elaspsed` typo to `Elapsed` since we're here.

Reply:
Fixed.